aten/src/ATen/native/RNN.cpp

```diff
@@ -747,7 +747,48 @@ std::tuple<Tensor, Tensor> NAME(                                               \
   return std::make_tuple(packed_output.data, std::get<1>(result));             \
 }
 
+#define ONE_HIDDEN_QRNN(NAME, CELL)                                             \
+std::tuple<Tensor, Tensor> NAME(                                               \
+      const Tensor& _input, const Tensor& hx,                                  \
+      TensorList _params, bool has_biases,                                     \
+      int64_t num_layers, double dropout_p, bool train, bool bidirectional, bool batch_first) { \
+  if (at::cudnn_is_acceptable(_input)) {                                       \
+    Tensor output, hy;                                                         \
+    gru_cudnn_stub(_input.type().device_type(), output, hy, _input, hx, _params, has_biases, \
+            num_layers, dropout_p, train, bidirectional, batch_first);         \
+    return std::make_tuple(output, hy);                                        \
+  }                                                                            \
+  check_device(_input, _params, hx);                                        \
+  auto input = batch_first ? _input.transpose(0, 1) : _input;                  \
+  auto params = gather_quantized_params(_params);                            \
+  auto results = _rnn_impl_with_concat<CELL, FullLayer, FullBidirectionalLayer>( \
+          input, params, hx.unbind(0), num_layers, dropout_p, train, bidirectional); \
+  if (batch_first) {                                                           \
+    std::get<0>(results) = std::get<0>(results).transpose(0, 1);               \
+  }                                                                            \
+  return results;                                                              \
+}                                                                              \
+                                                                               \
+std::tuple<Tensor, Tensor> NAME(                                               \
+      const Tensor& data, const Tensor& batch_sizes, const Tensor& hx,         \
+      TensorList _params, bool has_biases,                                     \
+      int64_t num_layers, double dropout_p, bool train, bool bidirectional) {  \
+  if (at::cudnn_is_acceptable(data)) {                                         \
+    Tensor output, hy;                                                         \
+    gru_packed_cudnn_stub(data.type().device_type(), output, hy, data, batch_sizes, hx, \
+            _params, has_biases, num_layers, dropout_p, train, bidirectional); \
+    return std::make_tuple(output, hy);                                        \
+  }                                                                            \
+  PackedSequence input { data, batch_sizes };                                  \
```

Comment:
```suggestion
  PackedSequence input = {data, batch_sizes};                                  \
```
https://abseil.io/tips/88